cirq/sim/simulator_test.py

```diff
@@ -314,6 +316,19 @@ def test_simulation_trial_result_qubit_map():
     assert result.qubit_map == {q[0]: 0, q[1]: 1}
 
 
+def test_verify_unique_measurement_keys():
+    q = cirq.LineQubit.range(2)
+    circuit = cirq.Circuit()
+    circuit.append([
+        cirq.measure(q[0], key='a'),
+        cirq.measure(q[1], key='a'),
+        cirq.measure(q[0], key='b'),
+        cirq.measure(q[1], key='b')
+    ])
+    with pytest.raises(ValueError, match='Measurement key a,b repeated'):
+        cirq.sample(circuit)
```

Comment:
Change to `_ = cirq.sample(circuit)` to signal that what's returned here is deliberately ignored.